src/sagemaker/inputs.py

```diff
@@ -136,6 +136,62 @@ class CreateModelInput(object):
     accelerator_type: str = attr.ib(default=None)
 
 
+@attr.s
+class CompilationInput(object):
```

Comment:
Is there any test, to verify the functionality of this class?

Reply:
Added a unit test